### PR TITLE
changes made to js file mainly

### PR DESCRIPTION
**Pull request by riya461**
I have made changes to js file.

### CODE REVIEW

**Comment by AnkitaMalik22** [comment]
Few more changes needed -
- texts not visible in dark mode
- decrease the font size
- fix the margin / padding and keep it as it was before in the site.

**Comment by riya461** [comment]
I have made new changes as to work in dark mode too

File: styles/Home.module.css [hunk 1]
```diff
@@ -57,10 +57,10 @@
 }
 
 .code {
-  color: black;
+  background: #fafafa;
```
Comment (AnkitaMalik22) [inline styles/Home.module.css:60]: remove background

Comment (AnkitaMalik22) [reply styles/Home.module.css:60]: and set 
`margin: 2rem 0;`
at **description** class

**Review by AnkitaMalik22 — CHANGES_REQUESTED** [review]

**Review by AnkitaMalik22 — APPROVED** [review]

**Comment by AnkitaMalik22** [comment]
Thanks for Contributing @riya461

**Pull request merged**